core/src/main/java/org/secondbase/core/SecondBase.java

```diff
@@ -43,6 +54,27 @@ private void init(final String[] args, final Flags flags) {
             flags.printVersion(System.out);
             System.exit(0);
         }
+
+        if (webConsole != null) {
+            try {
+                webConsole.start();
+            } catch (final IOException e) {
+                throw new SecondBaseException("Could not start webconsole.", e);
+            }
+        }
+
+        // Shut down modules on kill command
+        Runtime.getRuntime().addShutdownHook(new Thread() {
+            public void run() {
+                if (webConsole != null) {
+                    try {
+                        webConsole.shutdown();
+                    } catch (final IOException e) {
+                        LOG.log(Level.SEVERE, "Could not shutdown webconsole.", e);
```

Comment:
From my experience the log message will not reach the console from the shutdown hook. I suggest to print the exception message to standard error.